python/py-urllib3/Portfile

```diff
@@ -26,6 +26,10 @@ checksums           rmd160  8e110cf646403b479129233ce7c16cff4f935576 \
 if {${name} ne ${subport}} {
     depends_build-append \
                         port:py${python.version}-setuptools
+    depends_lib-append \
+                        port:py${python.version}-brotli \
+                        port:py${python.version}-certifi \
+                        port:py${python.version}-ipaddress
```

Comment:
Specified dependencies are marked as 'extra' for urllib3 module:
```
extras_require={
        "brotli": ["brotlipy>=0.6.0"],
        "secure": [
            "pyOpenSSL>=0.14",
            "cryptography>=1.3.4",
            "idna>=2.0.0",
            "certifi",
            "ipaddress; python_version=='2.7'",
        ],
        "socks": ["PySocks>=1.5.6,<2.0,!=1.5.7"],
    },
)
```

(note: ipaddress is python27 only)

So I think it'll be better to add those dependencies directly to py-hggit port.

Reply:
Good point, although `certifi` and the `secure` extra in general does seem reasonable to elevate, don't you think? (Accessing HTTPS URLs is pretty standard these days…)

Reply:
Defaulting to adding https support seems like a good idea to me.

Reply:
I moved the `brotli` dependency to `hg-git`.